src/main/scala/com/github/ahnfelt/react4s/Css.scala

```diff
@@ -3,109 +3,140 @@ package com.github.ahnfelt.react4s
 import scala.collection.mutable.ListBuffer
 
 trait CssChild
-case class CssPseudoClass(name : String, children : Seq[CssChild]) extends CssChild
-case class CssMediaQuery(query : String, children : Seq[CssChild]) extends CssChild
-case class CssSelector(selector : String, children : Seq[CssChild]) extends CssChild
+case class CssPseudoClass(name: String, children: Seq[CssChild])
+    extends CssChild
+case class CssMediaQuery(query: String, children: Seq[CssChild])
+    extends CssChild
+case class CssSelector(selector: String, children: Seq[CssChild])
+    extends CssChild
 
 object Css {
 
-    def selector(selector : String, children : CssChild*) = CssSelector(selector, children)
-    def media(query : String, children : CssChild*) = CssMediaQuery(query, children)
-    def pseudo(pseudoClass : String, children : CssChild*) = CssPseudoClass(pseudoClass, children)
-    def cssClass(className : String, children : CssChild*) = CssSelector("." + className, children)
-    def directChild(tagName : String, children : CssChild*) = CssSelector(">" + tagName, children)
-    def mediaSmall(children : CssChild*) = CssMediaQuery("(max-width: 991px)", children)
-    def mediaLarge(children : CssChild*) = CssMediaQuery("(min-width: 992px)", children)
+  def selector(selector: String, children: CssChild*) =
```

Comment:
Why did scalafmt break some of these lines?

Reply:
I am not sure - it must be part of the default rules.

Reply:
Ah I had the config file in the wrong spot. Pushed a change - see if you like it better.

Reply:
We can tweak it as much as we like:
https://scalameta.org/scalafmt/docs/configuration.html